README.md

```diff
@@ -1,2 +1,134 @@
-# vonage-spring-boot-starter-java
-Spring Boot Starter for Vonage
+# vonage Spring Boot Starter
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.nexmo/nexmo-spring-boot-starter.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.nexmo%22%20AND%20a:%22nexmo-spring-boot-starter%22)
+[![Build Status](https://travis-ci.org/Nexmo/nexmo-spring-boot-starter.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-spring-boot-starter)
+[![codecov](https://codecov.io/gh/Nexmo/nexmo-spring-boot-starter/branch/master/graph/badge.svg)](https://codecov.io/gh/Nexmo/nexmo-spring-boot-starter)
+
+<img src="https://developer.nexmo.com/assets/images/Vonage_Nexmo.svg" height="48px" alt="Nexmo is now known as Vonage" />
+
+This Spring Boot Starter has been provided to help with integrating the [Nexmo Java SDK](https://github.com/Nexmo/Nexmo-java) into your [Spring Boot](https://spring.io/projects/spring-boot) project.
+
+ * [Installation](#installation)
+ * [Usage](#usage)
+ * [Customizing the VonageClient](#customizing-the-vonageclient)
+ * [Customize vonage Client Version](#customize-vonage-client-version)
+ 
+## Installation
+
+For Gradle:
+
+```groovy
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.vonage:vonage-spring-boot-starter:1.1.0'
```

Comment:
Change `com.vonage:vonage-spring-boot-starter:1.1.0` to `com.vonage:vonage-spring-boot-starter:1.0.0`